nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/SplitText.java

```diff
@@ -134,6 +168,28 @@ protected void init(final ProcessorInitializationContext context) {
     }
 
     @Override
+    protected Collection<ValidationResult> customValidate(ValidationContext validationContext) {
+        ArrayList<ValidationResult> results = new ArrayList<>();
```

Comment:
Is there any reason we are declaring the type here as ArrayList, rather than List?